CometServer/Services/Operations/SideEffects/Implementation/ParameterSideEffect.cs

```diff
@@ -74,6 +77,16 @@ public sealed class ParameterSideEffect : OperationSideEffect<Parameter>
         /// </summary>
         public ICachedReferenceDataService CachedReferenceDataService { get; set; }
 
+        /// <summary>
+        /// Gets or sets the <see cref="IParameterService"/>
+        /// </summary>
+        public IParameterService ParameterService { get; set; }
+
+        /// <summary>
+        /// Gets or sets the <see cref="IParameterTypeService"/>
+        /// </summary>
+        public IParameterTypeService ParameterTypeService { get; set; }
```

Comment:
i think you should be using `ICachedReferenceDataService` and `CachedReferenceDataService`